plugins/database/mssql/mssql-database-plugin/main.go

```diff
@@ -0,0 +1,16 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/hashicorp/vault/plugins/database/mssql"
+)
+
+func main() {
+	err := mssql.Run()
+	if err != nil {
+		fmt.Println(err)
```

Comment:
If I switch this to a log.Println() it comes out in the logs like:
```
2017/05/03 12:09:12 [DEBUG] plugin: mysql-database-plugin: 2017/05/03 12:09:12 TEST
```
And the CLI spits out this error:

```
Error writing data to database/config/mysql: Error making API request.

URL: PUT http://127.0.0.1:8200/v1/database/config/mysql
Code: 400. Errors:

* error creating database object: plugin exited before we could connect
```